package.json

```diff
@@ -18,7 +18,7 @@
     "re:format": "rescript format -all",
     "test:start": "node dist/server/server.js",
     "postinstall": "git config core.hooksPath .githooks && chmod +x .githooks/commit-msg",
-    "cy:open": "sh cypress/start_hyperswitch.sh && cypress open",
+    "cy:open": "sh cypress/start_hyperswitch.sh && cypress run",
     "cy:run": "sh cypress/start_hyperswitch.sh && cypress run",
```

Comment:
cy:open and cy:run both will trigger the same command, are both required ?

Reply:
updated